test/run

```diff
@@ -44,7 +44,13 @@ main() {
         write out
         quit!
       "
-      ${test}/../src/kak out -n -u -e "$kak_commands"
+      if [ -z ${TEST_LOAD_KAKRC+x} ]; then
```

Comment:
Yeah it is wonky I agree. That condition is true only if TEST_LOAD_KAKRC is effectively not set, but fails if TEST_LOAD_KAKRC is set and empty. +x is just a random thing, could have been +anything (http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02).

The problem is that I don't know what variable would be sensible to use in this case. Using the actual -d switch as its value sounds ugly.
The biggest problem though is that it doesn't even seem to work on travis as it is now.